report_qweb_table_pagebreak/models/ir_qweb.py

```diff
@@ -0,0 +1,104 @@
+from lxml import html
+
+from odoo import api, models
+
+from odoo.addons.base.models.qweb import QWeb
+
+
+class StyleParser:
+    """Class to manipulate and Serialize/DeSerialize style attribute on
+    html tag::
+
+        style = StyleParser(element.get("style"))
+        # set attribute page-break-inside
+        style.set("page-break-inside", "avoid")
+        # Serialize style to set style value on html element
+        element.set("style", str(style))
+    """
+
+    _styles: dict = {}
+
+    def __init__(self, style_content: str):
+        self._parse(style_content)
+
+    def _parse(self, style_content):
+        self._styles = {}
+        for style in style_content.split(";"):
+            data_style = style.split(":")
+            if len(data_style) != 2:
+                # ignore if not expected string
+                continue
+            key, value = data_style
+            self._styles[key.strip()] = value.strip()
+
+    def __str__(self):
+        return "; ".join([f"{k}: {v}" for k, v in self._styles.items()])
+
+    def set(self, key, value, default=None):
+        if value is None:
+            value = default
+        if value is None:
+            self._styles.pop(key, None)
+        else:
+            self._styles[key] = value
+
+
+class IrQWeb(models.AbstractModel, QWeb):
+    _inherit = "ir.qweb"
+    _description = "Qweb"
```

Comment:
```suggestion
```
Why overriding the `_description` of the base model with the same value?

Reply:
:pray: too large copy/past

Reply:
done within 348eacfe